network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/waitlist/NetPWaitlistManager.kt

```diff
@@ -52,7 +53,7 @@ interface NetPWaitlistManager {
 @SingleInstanceIn(AppScope::class)
 @ContributesBinding(AppScope::class)
 class RealNetPWaitlistManager @Inject constructor(
-    private val service: WgVpnControllerService,
+    @ProtectedVpnControllerService private val service: WgVpnControllerService,
```

Comment:
Ditto, ie. this probably can also rather be `@UnprotectedVpnControllerService` at which point we delete the `@ProtectedVpnControllerService` altogether